### ISSUE TITLE
Many through another many

### ISSUE THREAD

**Issue by Aryk** [tommikaikkonen/redux-orm#72]
I know you can do through associations where the central table is a join table and you have two foreign keys, but what if you want to two levels on a one to many. You could just map the result and create a query set from that, but is it doable with redux-orm library?

**Comment by tommikaikkonen** [comment]
You mean that following a foreign key `A.F1` on the join table and getting a set of objects `[X]` that match the foreign key, you want to do that and then additionally follow the foreign keys in each `Xn.F2` to get another set of objects `[Y]` and make that into a single lookup `A.F1 -> [Y]`?

Correct, there's no direct support for that in redux-orm. There's no API on the ORM side implemented for that and the database would need to do multiple joins on a single query, which it can not at this point. You could implement a method (or a property getter if it doesn't overlap with field names) on the Model instances where you need to do the lookup to hide the manual concat-mapping that is needed.

**Comment by Aryk** [comment]
Yes, that's I think we are on the same page. I guess that way I solved this is to map the other association, flatten the array, and throw it into a query set, so 

City =-> cars => passengers

```javascript
class City {
  function passengers {
    new QuerySet(flatten(this.cars.toModelArray().map(c => c.passengers)))
  }
}

```

was my solution.

That way I can keep the query set chaining alive.
